bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,60 @@
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours
-from bot.decorators import override_in_channel
-
 log = logging.getLogger(__name__)
 
+icons = {"issue": "https://i.imgur.com/HFV0nv9.png",
+         "issue-closed": "https://i.imgur.com/uZSX7as.png",
+         "pull-request": "https://i.imgur.com/zHhnALX.png",
+         "pull-request-closed": "https://i.imgur.com/JYmTn5P.png",
+         "merge": "https://i.imgur.com/xzQZxXe.png"}
```

Comment:
Once uploaded to the emoji server these need to be added to `Constants.py` rather than being hardcoded into the cog.

Upload is also pending approval from GH to utilize modified versions of their icons. While GH [provides their Octicons](https://octicons.github.com/) there is a [clause potentially specifying that the colors may not be adjusted](https://github.com/logos). However, it is not clear whether this applies only to their logos or if it applies broadly to all of their icons; a request for clarification has been sent to GH.

Reply:
Agreed, this is temporary as agreed in dev chat.